env.sh

```diff
@@ -7,3 +7,4 @@ git config --global --add safe.directory /c/Users/davidflanderka/vyvoj/flow123d/
 git config --global --add safe.directory /c/Users/davidflanderka/vyvoj/flow123d/third_party/bparser
 git config --global --add safe.directory /c/Users/davidflanderka/vyvoj/flow123d/third_party/gtest-1.10.0
 git config --global --add safe.directory /c/Users/davidflanderka/vyvoj/flow123d/third_party/json-3.10.5
+git config --global --add safe.directory /c/Users/davidflanderka/vyvoj/flow123d/third_party/pybind11-2.9.2
```

Comment:
- merge master
- check functionality: `make set-safe-directory` 
- add pybind11 into the make rule

Reply:
Script env.sh calls `make set-safe-directory` rule instead of repeated calls of `git config --global ...` for each directory